src/project-2.js

```diff
@@ -3,6 +3,17 @@
 const getBiggest = (x, y) => {
   // x and y are integers.  Return the larger integer
   // if they are the same return either one
+  if (x === y) {
```

Comment:
While these logic conditions work, there may be a more effective/efficient way to solve this .. :)

Reply:
we could omit that whole if statement and return either x and y at the end since the other tests have eliminated all other cases.